pkg/log/status.go

```diff
@@ -196,18 +196,22 @@ func Askf(format string, a ...interface{}) {
 // Status creates a spinner, sets the prefix then returns it.
 // Remember to use .End(bool) to stop the spin / when you're done.
 // For example: defer s.End(false)
-func Spinner(status string) *Status {
+// Spin parameter is used to start the initial spin. This helpful
+// for situations where spinning isn't viable (debug)
+func Spinner(spin bool, status string) *Status {
```

Comment:
I would rather add another `SpinnerWith(status string, force bool)` function the original `Spinner` function would call so that client code already using `Spinner` doesn't change. Also, generally speaking, the `bool` argument is less important than the `status` string so I'd place it second. 
Another option (if adding a function is too much work ;)) would be to add a variadic arg to simulate optional argument for the `bool` argument, though that wouldn't work with the `Spinnerf` version (which should also be changed, imo).

Reply:
I would like to avoid variadic arg unless they are absolutely necessary to tell you the truth. IMHO the hinder readability

Reply:
@metacosm I'd agree with adding a separate function, if we don't want a "spinning" spinner it should be separate rather than relying on the boolean value.